.buildkite/pipeline.yml

```diff
@@ -55,7 +55,7 @@ steps:
 #  - name: "Codegen"
 #    env:
 #      CGO_ENABLED: 0
-#      GIMME_GO_VERSION: 1.18.x
+#      GIMME_GO_VERSION: 1.22.x
```

Comment:
can you add these things to the common env decl above (line 49 in this file)

Reply:
done!